airlock/issues.py

```diff
@@ -36,6 +37,16 @@ def ensure_issue_status_labels(org, repo, github_api):
     return repo_labels
 
 
+def get_github_max_retry():
+    return settings.DEFAULT_MAX_GITHUB_RETRIES
```

Comment:
Why do we need a function to just return the setting?  When it was used as `max_tries` in the `backoff` library, it could be a callable, but didn't have to be.  Now it's just being called in `stamina.retry`, so I don't think the function adds anything useful?